pkg/operator/reports/name.go

```diff
@@ -0,0 +1,36 @@
+package reports
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/aquasecurity/trivy-operator/pkg/kube"
+	"k8s.io/apimachinery/pkg/util/validation"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func NameFromController(controller client.Object) string {
```

Comment:
I would change the func name to be more clear to `NameController`

Reply:
Usually I try to name methods by how they are used, and I think `reports.NameFromController` reads better than `reports.NameController`. Remember that the parameter is just `client.Object`. It would be even better if the package wasn't named with plural (`report`), but that breaks linting.

Reply:
Yeah, also prefer `NameFromController`, `NameController` doesn't give the feeling a name will be returned, maybe if we add an action before like `CreateNameFromController`? wdyt?